estate/models/estate_property.py

```diff
@@ -0,0 +1,105 @@
+from odoo import api, fields, models
+from dateutil.relativedelta import relativedelta
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_is_zero, float_compare
+
+
+class propertiesPlan(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Properties Plan"
+    _order = "id desc"
+
+    name = fields.Char("Name", required=True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date("Available Date", copy=False, default=fields.Datetime.today() + relativedelta(months=3))
+    expected_price = fields.Float("Expected Price", required=True)
+    selling_price = fields.Float("Selling Price", readonly=True, copy=False)
+    bedrooms = fields.Integer("Bedrooms", default=2)
+    living_area = fields.Integer("Living Area (sqm)")
+    facades = fields.Integer("Facades")
+    garage = fields.Boolean("Garage")
+    garden = fields.Boolean("Garden")
+    garden_area = fields.Integer("Garden Area (sqm)")
+    buyer = fields.Char("Buyer", copy=False)
+    total_area = fields.Integer("Total Area (sqm)", compute="_compute_total_area")
+    best_price = fields.Float("Best Price", compute="_compute_best_price")
+    sales_person = fields.Many2one(
+        'res.users',
+        default=lambda self: self.env.user or False,
+        string="Sales Person"
+    )
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')],
+        help="Select the direction to filter the options",
+    )
+    active = fields.Boolean("Active", default=True)
+    state = fields.Selection(
+        string='State',
+        selection=[('new', 'New'), ('offer_received', 'Offer Received'), ('offer_accepted', 'Offer Accepted'), ('sold', 'Sold'), ('canceled', 'Canceled')],
+        help="State of your property.",
+        default='new'
+    )
```

Comment:
```suggestion
    state = fields.Selection(
        string='State',
        selection=[
            ('new', 'New'), 
            ('offer_received', 'Offer Received'), 
            ('offer_accepted', 'Offer Accepted'), 
            ('sold', 'Sold'), ('canceled', 'Canceled')
        ], help="State of your property.", default='new')
```
For readability and in order not to change the whole line to add/remove on